docs/reference/slm/slm-retention.asciidoc

```diff
@@ -46,35 +47,7 @@ PUT /_slm/policy/daily-snapshots
 <2> Keep snapshots for 30 days
 <3> Always keep at least 5 successful snapshots
 <4> Keep no more than 50 successful snapshots
-
-Supported configuration for retention from within a policy are as follows. The default value for
-each is unset unless specified by the user in the policy configuration.
-
-NOTE: The oldest snapshots are always deleted first, in the case of a `max_count` of 5 for a policy
-with 6 snapshots, the oldest snapshot will be deleted.
-
-|=====================================
-| Setting | Description
-| `expire_after` | A timevalue for how old a snapshot must be in order to be eligible for deletion.
-| `min_count` | A minimum number of snapshots to keep, regardless of age.
-| `max_count` | The maximum number of snapshots to keep, regardless of age.
```

Comment:
We're removing the description of these options, but I don't see them documented anywhere else now other than once in the example API request. I think we still need to document these configuration options somewhere

Reply:
They are in the API reference: https://github.com/elastic/elasticsearch/pull/55953/files#diff-3e796bf77f9921d05504be8c3e5a9349